### DISCUSSION TITLE
How do I convert `Vec<Value>` to and from `Vec<Response>`

### DISCUSSION THREAD

**Discussion by ShayBox** [SeaQL/sea-orm#1317] [Q&A] [answered]
I'm saving and loading a struct `Response` from https://github.com/kiwiyou/craftping/blob/master/src/entity.rs#L20  
(The fork I'm using has `Deserialize`/`Serialize`)

How do I convert `Vec<Value>` to and from `Vec<Response>`?
```rust
    // How do I convert this into Vec<Response> from Vec<Value>
    let responses = server.responses.to_owned();
    // responses.push(response);
    println!("{:#?}", responses);

    let mut server: ActiveModel = server.into();
    // How do I convert this back into Vec<Value> from Vec<Response>?
    server.responses = Set(responses);
    // server.updated_at = Set();
    server.update(db).await?;
```
Also how do I set the `updated_at` to the current `DateTime` (PostgreSQL Timestamp)

**Comment by billy1624** [comment, marked as answer]
Hey @ShayBox, which `Vec<Value>` do you mean? `Vec<sea_query::Value>` or `Vec<serde_json::Value>`?

**Reply by ShayBox** [reply]
`pub responses: Vec<Json>` `serde_json::Value`

**Reply by billy1624** [reply]
This is outside the scope of sea-orm.

You need this method: https://docs.rs/serde_json/latest/serde_json/fn.from_value.html

**Comment by billy1624** [comment]
> Also how do I set the `updated_at` to the current `DateTime` (PostgreSQL Timestamp)

Currently, we only support setting `sea_query::Value` in `ActiveModel`. We're investigating the possible solution to allow setting expression i.e. `sea_query::SimpleExpr` in it.
